src/test/java/core/basesyntax/serviceimpl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.serviceimpl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.ParserService;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    @BeforeAll
+    static void clear() {
+        Storage.storage.clear();
+    }
+
+    @AfterEach
+    void clearAfter() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    void when_NegativeAmount_NotOk() {
+        List<String> list = List.of("b,apple,-700");
+        ParserService parserService = new ParserServiceImpl();
```

Comment:
You are creating new ParserServiceImpl in every test, you can make @BeforeEach and do it there, same logic can be implemented in the rest test classes